src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -807,11 +805,18 @@ def extract_assay_germline(args):
                     if r["genotype_type"] == "hom":
                         r["genotype_type"] = "hom-alt"
 
+            def sort_variant(d):
+                chrom, pos = d["allele_id"].split("_")[:2]
+                if chrom.isdigit():
+                    return int(chrom), '', int(pos)
+                return float('inf'), chrom, int(pos)
+            ordered_results = sorted(resp_raw["results"], key=sort_variant)
```

Comment:
Did you run e2e tests (in pseudopod GH actions)  using this dx-toolkit branch (only this code, not for all apps)? I am afraid that will fail in some cases - so we are not surprised when it will get released.